autoload/phpactor.vim

```diff
@@ -10,8 +10,11 @@
 " NOTE: This help is auto-generated from the VimScript using
 "     https://github.com/google/vimdoc. See
 "     https://phpactor.github.io/phpactor/developing.html#vim-help
+" vint: -ProhibitUnnecessaryDoubleQuote
+" vint: -ProhibitImplicitScopeVariable
+" vint: -ProhibitAbbreviationOption
```

Comment:
The amount of changes required to fix all these is too much of a risk (I tried fixing them all manually, but things broke immediately).

Reply:
> " vint: -ProhibitUnnecessaryDoubleQuote

JFI: this check is actually too strict/noisy, see https://github.com/Vimjas/vint/issues/310.

Reply:
btw: in general I recommend to try/use vint master (also).